### DISCUSSION TITLE
Discord community call: 9 December 2023

### DISCUSSION THREAD

**Discussion by BenJamesBen** [freesewing/freesewing#5530] [Support] [answered]
You are all invited to our Discord hangout.
Please leave a comment here or [on Discord](https://discord.freesewing.org/) to add things to the agenda.
This page will be updated after the call with notes.

### About

FreeSewing's Discord hangout is an informal hangout/chat/meeting that is open to everyone.
The call is organized/hosted by @tangerineshark.
The call does not have a fixed structure or agenda items. Everyone is free to suggest agenda items, ask questions, or bring ideas.

### Details

Location: https://discord.freesewing.org/ (meeting room: 🔊voice chat)
Date: December 9th, 2023 5pm UTC, which can be found in your local time zone [here](https://savvytime.com/converter/utc-to-belgium-antwerpen-ca-los-angeles-il-chicago-ny-new-york-city-brazil-sao-paulo-united-kingdom-london-france-paris-south-africa-johannesburg-msd-india-mumbai-hkt-japan-tokyo-australia-sydney/dec-9-2023/5pm).
Or check the Events tab in Discord!
The meetings will always be at 6pm, Antwerp time.

### Agenda

- Additions/updates to Frequently Asked Questions (see details below)
- Do we still want to hold our December 23, 2023 meeting or should it be cancelled?

**Comment by nataliasayang** [comment]
I'd like to propose an agenda item: **Additions/updates to Frequently Asked Questions**

Additions:
- Where did the "standard" measurements go?
- Why are seam allowances not included by default?

Update:
- I want to check if there should be two newsletter FAQs, or if they can be combined into one.

I am of course willing to do the actual updates, but want to discuss whether it's appropriate to make these changes and get the "official" answers.

**Comment by tangerineshark** [comment]
I won't be available for this call, but that shouldn't hold anything up! Just wanted to give a heads up!

**Comment by BenJamesBen** [comment, marked as answer]
Meeting Notes:

- For the proposed FAQ additions and updates she wanted to make, Natalia will just ping Joost for go-ahead feedback.
- We decided that the next scheduled meeting, December 23, 2023, will not be an official meeting. Instead we will hold an informal social hangout call, perhaps with gaming towards the end.
 
Other discussion:
- Working on the Opal design and a new scromper pattern.
- Best practice for a FreeSewing design with different seam allowances or sections with multiple folds for reinforcement? Suggestion: use a different line and add it to the pattern, rather than using the Seam Allowance value.
- Don't feel guilty for not participating as much in FreeSewing as you might want.
- Watching YouTube sewing videos to get back into sewing, TheClosetHistorian channel.
- Working on 3D body scans to get an accurate 3D avatar for garment making. Link to an academic paper about body scans using WiFi signals.
- Working on an ostomy belt which needs strong fabric. Suggestions: woven material like bobbinet, swim fabric, perhaps a modified Onyx.
- There is snow in Europe right now which is earlier than usual.
- Made a quilt shop haul of fat quarters to make a quilt.
- Planning to make a new corset pattern. 1890's Victorian corset? S-bend corset?
- Working on a legging pattern.
- Incorporating Skully feedback from someone who made one.
- Making tiny Skully's and Hi sharks. Making clothing out of one or more Hi sharks.
- Designing a Vampirella costume.
- An issue where the FreeSewing UI defaults back to control level 2 and back to Dutch when the web page is reloaded?
- Showing off work on the Opal bib overall design.
